lms/djangoapps/discussion_api/api.py

```diff
@@ -43,15 +43,15 @@
 from openedx.core.djangoapps.course_groups.cohorts import get_cohort_id
 
 
-def _get_course_or_404(course_key, user):
+def _get_course_or_raise_not_found(course_key, user):
```

Comment:
I know there are a couple of Django method which use this naming convention but I think just `_get_course` should be sufficient.